src/mame/tvgames/spg2xx.cpp

```diff
@@ -1851,6 +1873,26 @@ void spg2xx_game_lexiart_state::lexiart(machine_config &config)
 }
 
 
+void spg2xx_game_ddr33v_state::init_ddr33v()
+{
+	// what is this checking? timer? battery state? protection? it goes to a blank screen after the boot logo otherwise
+	uint16_t* rom = (uint16_t*)memregion("maincpu")->base();
+	rom[0x208055] = 0x4440;
+}
+
+void spg2xx_game_ddr33v_state::ddr33v(machine_config &config)
+{
+	SPG24X(config, m_maincpu, XTAL(27'000'000), m_screen);
+	m_maincpu->set_addrmap(AS_PROGRAM, &spg2xx_game_ddr33v_state::mem_map_4m);
+
+	spg2xx_base(config);
+
+	m_maincpu->porta_in().set(FUNC(spg2xx_game_ddr33v_state::base_porta_r));
+	m_maincpu->portb_in().set(FUNC(spg2xx_game_ddr33v_state::base_portb_r));
+	m_maincpu->portc_in().set(FUNC(spg2xx_game_ddr33v_state::base_portc_r));
+}
```

Comment:
Isn’t this identical to `spg2xx_game_state::spg2xx`?

Reply:
well yes, but I think you're just showing ignorance of the reverse engineering process again.

additions here start by copying the boilerplate, this makes it easy to customize / research - in the case of this one, whatever the thing I've ended up patching out is, but ultimately after trying a few things I decided on a patch for now and the config just ended up being the same.

it's was left a timesaver for whoever comes next, but fine, it can go.